app/setting/apikey/apiKey.tsx

```diff
@@ -64,8 +63,7 @@ function ApiKey() {
                 />
               )}
             </Head>
-            {loading && <ApiKeyLoading />}
-            {!loading && isEmpty(apiKeys) && (
+            {isEmpty(apiKeys) && (
```

Comment:
We will not remove loading from here because it's use for page loading.
We will set condition for display loading, empty state or list same as we do in other pages.